Mixpanel/MPUITableViewBinding.m

```diff
@@ -122,4 +122,18 @@ - (UITableView *)parentTableView:(UIView *)cell {
     return nil; // this view is not within a tableView
 }
 
+- (BOOL)isEqual:(id)other {
```

Comment:
I don't see an scenario where that could happen. But if you feel more comfortable leaving this I am fine.